cmake/std/PullRequestLinuxDriver.sh

```diff
@@ -123,23 +198,44 @@ echo -e "PRDriver> "
 echo -e "PRDriver> Driver and Merge scripts unchanged, proceeding to TEST phase"
 echo -e "PRDriver> "
 
+# determine what MODE we are using
+mode="standard"
+if [[ "${JOB_BASE_NAME}" == "Trilinos_pullrequest_gcc_8.3.0_installation_testing" ]]; then
+    mode="installation"
+fi
+
+
+
 # Prepare the command for the TEST operation
 test_cmd_options=(
-    ${TRILINOS_SOURCE_REPO:?}
-    ${TRILINOS_SOURCE_BRANCH:?}
-    ${TRILINOS_TARGET_REPO:?}
-    ${TRILINOS_TARGET_BRANCH:?}
-    ${JOB_BASE_NAME:?}
-    ${PULLREQUESTNUM:?}
-    ${BUILD_NUMBER:?}
-    ${WORKSPACE:?}
-    )
-test_cmd="${SCRIPTPATH}/PullRequestLinuxDriverTest.py ${test_cmd_options[@]}"
+    --sourceRepo=${TRILINOS_SOURCE_REPO:?}
+    --sourceBranch=${TRILINOS_SOURCE_BRANCH:?}
+    --targetRepo=${TRILINOS_TARGET_REPO:?}
+    --targetBranch=${TRILINOS_TARGET_BRANCH:?}
+    --job_base_name=${JOB_BASE_NAME:?}
+    --workspaceDir=${WORKSPACE:?}
+    --github_pr_number=${PULLREQUESTNUM:?}
+    --job_number=${BUILD_NUMBER:?}
+    --req-mem-per-core=3.0
+    --max-cores-allowed=29
+    --num-concurrent-tests=4
+    --mode=${mode}
+    #--mode=installation
+    --config="Trilinos/cmake/std/pr_config/pullrequest.ini"
+    #--dry-run
```

Comment:
Your command line options have multiple styles:  `--fooBar`, `--foo_bar`, and `--foo-bar`.  I think the third is the preference in the community.

Reply:
Yep. 

I generally prefer using dashes like `--foo-bar` ... I will look at this with some cleanup work.

I think I kept these as is because some of it was there to match what was in the original script that I ported over from.  It was easier to do just keeping things the same.

Now that it's working in the PR system, it'll be easier to do a search/replace ;)

I'll add this to my list of todo items.